tests/dummy/app/application/route.js

```diff
@@ -1,33 +1,6 @@
 import Route from '@ember/routing/route';
-import { debug } from '@ember/debug';
 import ENV from '../config/environment';
 export default Route.extend({
-  beforeModel () {
```

Comment:
cool - I'm happy to see you took this on, as I was going to get to it when I got to it in another PR - were you able to test this in IE? Did it work?

Reply:
> were you able to test this in IE? Did it work?

yup. it appears we're polyfilling _all the time_ in the dummy app now.